adopters.md

```diff
@@ -10,7 +10,7 @@ This is a list of TiDB adopters in various industries.
 
 | Company | Industry | Case studies |
 | :--- | :--- | :--- |
-|[SHAREit Group](https://www.crunchbase.com/organization/shareit-2)|Internet|[English](https://pingcap.com/case-studies/why-shareit-selects-tikv-for-data-storage-for-its-2.4-billion-user-business); [Chinese](https://pingcap.com/zh/blog/devcon-2021-shareit)|
+|[SHAREit Group](https://www.crunchbase.com/organization/shareit-2)|Internet|[English](https://pingcap.com/case-study/how-shareit-powers-its-ai-workflow-with-tidb-and-tikv/); [Chinese](https://pingcap.com/zh/blog/devcon-2021-shareit)|
```

Comment:
Just a reminder, if you find broken links on `pingcap.com/case-studies` or `pingcap.com/blog`, report these links to the blog team. They should add redirects for these links.